src/java/org/apache/cassandra/net/StartupClusterConnectivityChecker.java

```diff
@@ -213,4 +261,11 @@ boolean incrementAndCheck(InetAddressAndPort address)
             return acks.computeIfAbsent(address, addr -> new AtomicInteger(0)).incrementAndGet() == threshold;
         }
     }
+
+    private String fmtBlocker(Integer size, Integer count)
```

Comment:
Is this a more complicated Math.max?

Reply:
Heh, yea it is I just thought that it was slightly clearer what it was doing (so we don't print `Integer.MAX_VALUE` to the log. I'll change it to `Math.max` instead.

Reply:
I killed it.